src/page/MyShopPage/components/EditShopInfoModal/index.tsx

```diff
@@ -81,14 +86,24 @@ EditShopInfoModalProps) {
     resolver: zodResolver(OwnerShop),
   });
 
+  useEffect(() => {
+    if (imageFile && !uploadError) { // 초기에 이 값이 true기 때문에 imageUrls가 빈 배열로 초기화되고 있었음
+      setImageUrls(imageFile);
+    }
+  }, [imageFile, setImageUrls]);
+
   const mutation = useMutation({
     mutationFn: (form: OwnerShop) => putShop(shopInfo.id, form),
     onSuccess: () => {
       closeModal();
       setIsSuccess(true);
     },
+    onError: (e) => {
+      if (isKoinError(e)) {
+        showToast('error', e.message);
+      }
+    },
```

Comment:
```suggestion
      if (isKoinError(e)) {
        showToast('error', e.message);
        return;
      }
      
      sendClientError(e);
    },
```

실패시 처리 확인할 수 있게 올려주세요~~

Reply:
a6d9678152886b8011687b8fb1ceb8d7e3fbc3e3
굿~ 적용했습니다